library/ajax/get_fullscreen_pages.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/**
+ * get_fullscreen_pages.php
+ * This page responds to an AJAX request to retrieve the
+ * pages available for a particular fullscreen role.
+ * Returns an array of the available pages

```

Comment:
The above is good  Add the license and copyright information

Reply:
He has both here...

Reply:
This comment was in reference to the license and copyright stuff.  I was saying that "both license and copyright are here".  Maybe Terry meant other files that lack it.  Just make sure you put your stamp on stuff.

Reply:
I added those after this comment I think. Not sure why this file hasn't been "outdated"